mne_bids/config.py

```diff
@@ -55,6 +55,11 @@
                       'eeg': allowed_extensions_eeg,
                       'ieeg': allowed_extensions_ieeg}
 
+# allowed BIDS entities
+BIDS_ENTITIES = ('subject', 'session', 'task', 'acquisition',
+                 'processing', 'run', 'recording',
+                 'space', 'suffix', 'prefix')
```

Comment:
Are `suffix` and `prefix` actually BIDS **entities** in the strict sense?

Reply:
> Are suffix and prefix actually BIDS entities in the strict sense?

nope :)

Cool relevant PR which will make our BIDS-related life easier in the future: https://github.com/bids-standard/bids-specification/pull/475

Reply:
> nope :)

So… wondering if `BIDS_ENTITIES` is the best choice of a name here, then… how about `BIDS_PATH_ENTITIES`? ;)